### PR TITLE
Fix edge case in Cypress flaky tests "Cannot read properties of null …

### PR DESCRIPTION
**Pull request by sneridagh**
…(reading required)" when the schema action has not been fulfilled yet (for some reason) and the test tries to save.

I think I've found the cause of it... let's see. 

Another option would be that the reducer always return an empty object... @robgietema any reason why we opted to return null if the reducers are empty?

### CODE REVIEW

File: src/helpers/FormValidation/FormValidation.js [hunk 1]
```diff
@@ -349,7 +349,7 @@ class FormValidation {
     touchedField = {},
   } = {}) {
     return validateFieldsPerFieldset(
-      schema,
+      schema || {}, // If schema is null for some reason (schema request haven't fulfilled yet) we need to normalize it
```
Comment (davisagli) [inline src/helpers/FormValidation/FormValidation.js:352]: @sneridagh so if the schema hasn't loaded yet, we effectively just skip validation. Is that okay? I think so, since the validation that counts should happen on the server anyway.

**Review by davisagli — APPROVED** [review]

**Comment by davisagli** [comment]
Hmm, now there is a different error that sounds like it has the same cause:
```
CypressError: Timed out retrying after 5000ms: `cy.wait()` timed out waiting `5000ms` for the 1st request to the route: `schema`. No request ever occurred.`
```

So maybe the problem is that something is preventing requesting the schema in the first place?

**Comment by sneridagh** [comment]
@davisagli I've seen this one too, schemas that do not load (or that take far too long in the test?) No clue why could that be happening.

File: src/helpers/FormValidation/FormValidation.js (hunk 1 above)
Comment (sneridagh) [reply src/helpers/FormValidation/FormValidation.js:352]: yeah, maybe that is better.

**Review by davisagli — APPROVED** [review]

**Comment by sneridagh** [comment]
LGTM
